cypress/e2e/assetDuplicateLicence/audioLicenceDuplicate.cy.ts

```diff
@@ -0,0 +1,55 @@
+/// <reference types="cypress" />
+
+import { CY } from '../../utils/common'
+import { AUDIO_TYPES } from '../../utils/upload'
+let idLicence = ''
+const FILE_ID: Array<string> = []
+
+describe(`Test asset audio licence duplicate function, Env: ${CY.cfg}`,
+  { tags: '@assetAudioLicence', env: { visitBaseUrl: false } }, () => {
+    it('Prepare Test Data', ()=> {
+      cy.prepareData('audio/sample.mp3', 0)
+      cy.prepareData('audio/sample.wav', 0)
+      cy.prepareData('audio/sample.m4a', 0)
+    })
+    it('Cms-sys | CMS licence', () => {
+      cy.visit('/asset')
+      cy.api_waitPageLoad('licence-unknown')
+      idLicence = '100000'
```

Comment:
I think, each value that depends on Database, should be configurable. If you're going to repeat every value in every test, there will be hell to change the test data, run the tests in a different environment.

This issue appears on many places in your code (`cms32630`, ....). 
If your test depends on data. (UserId, ExtSystem) .... this should by configurable for test.
Additional data for testing e.g. different licence, you should store this data in constants. Then you should
1.  check if data exists
2. if there is no needed data, you should create it
3. after all, you use data in tests